lfs/upload_queue.go

```diff
@@ -34,6 +34,20 @@ func NewUploadable(oid, filename string, index, totalFiles int) (*Uploadable, *W
 	return &Uploadable{oid: oid, OidPath: path, Filename: filename, size: fi.Size()}, nil
 }
 
+func NewUploadableWithoutFilename(oid string) (*Uploadable, *WrappedError) {
```

Comment:
I think I'd rather have `NewUploadable` just respect a `""` filename, instead of a new function.